tsconfig.json

```diff
@@ -1,5 +1,6 @@
 {
   "compilerOptions": {
+    "target": "ES2020",
```

Comment:
I know nothing. Is this backwards compatible?
Does it unlock anything?

Reply:
Looks like it does some things. I did this originally to add some bigint features that weren't in the default version. https://www.learningtypescript.com/articles/why-increase-your-tsconfig-target